pheno_utils/bulk_data_loader.py

```diff
@@ -0,0 +1,48 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../nbs/14_bulk_data_loader.ipynb.
+
+# %% auto 0
+__all__ = ['FIELD_TYPE_TO_FUNC', 'get_function_for_field_type', 'load_image', 'show_fundus']
+
+# %% ../nbs/14_bulk_data_loader.ipynb 3
+import matplotlib.pyplot as plt
+from PIL import Image
+import warnings
+from smart_open import open
+import pandas as pd
+
+# %% ../nbs/14_bulk_data_loader.ipynb 4
+from pheno_utils.config import (
+    DICT_PROPERTY_PATH
+    )
+
+# %% ../nbs/14_bulk_data_loader.ipynb 5
+FIELD_TYPE_TO_FUNC = pd.read_csv(DICT_PROPERTY_PATH, index_col='field_type').to_dict()['load_func']
+
+# %% ../nbs/14_bulk_data_loader.ipynb 6
+def get_function_for_field_type(field_type):
+    function_name = FIELD_TYPE_TO_FUNC.get(field_type)
+    if function_name is not None:
+        return globals().get(function_name)
+    else:
+        # Handle the case where the field type is not found
+        return pd.read_parquet
```

Comment:
what should we do if the function name is not in the globals? it will return a None, and there will be a downstream error, but perhaps we should catch it here first ("field type load func is missing..." of sorts)

---

minor style suggestion (unless you object), instead of the if-else clause you may use:

function_name = FIELD_TYPE_TO_FUNC.get(field_type, 'read_parquet')

and have from pandas import read_parquet at the top of the py.

Reply:
Thanks! added to the new commit 
Please review :)